### ISSUE TITLE
iVim project wiki page

### ISSUE THREAD

**Issue by terrychou** [terrychou/iVim#82]
Hi, everyone,

I have decided to utilize the [project Wiki](https://github.com/terrychou/iVim/wiki) and been building it.

Information about how to properly use or config iVim will be put there. Please check it out.

If you have any suggestion, just let me know.

**Comment by htkm** [comment]
If I want to try something in the wiki, @terrychou do you have any preference for future inputs to be aligned in a specific way? Because I am new to here and want to follow your guideline.

**Comment by terrychou** [comment]
@htkm do you mean that you want to add something into the Wiki? If so, just try to focus one page on a specific topic with a clear title, so that we can locate the target page easily.

**Comment by htkm** [comment]
Yes. But I don't know how to `focus one page`. Where is it? Sorry for the repeating question.

**Comment by htkm** [comment]
OK, I found there's another git for wiki. Interesting.

**Comment by terrychou** [comment]
> Yes. But I don't know how to `focus one page`. Where is it? Sorry for the repeating question.

By this, I mean if you open a new page in the Wiki, rather than talking about many things, try just focus on one topic and give it a nice title. So that the users can know whether it is the page they need by a glance.

**Comment by htkm** [comment]
Gotcha. Literally focusing one topic with a clear title.

**Comment by sedm0784** [comment]
The discussion above makes it sound like you intended the Wiki to be [publicly editable](https://docs.github.com/en/github/building-a-strong-community/changing-access-permissions-for-wikis), but I don't seem to be able to do so. Can you change the access permissions, or if we want to write guides for iVim would you prefer them off-site with just a link in the [Writings from You issue](https://github.com/terrychou/iVim/issues/107)?

**Comment by terrychou** [comment]
@sedm0784 Thanks for this suggestion! I have removed the editing restriction of the Wiki and I think everyone can edit it now.
